plugins/ja3_fingerprint/ja3_fingerprint.cc

```diff
@@ -348,15 +361,14 @@ TSRemapInit(TSRemapInterface *api_info, char *errbuf, int errbuf_size)
 
   // Check if there is config conflict as both global and remap plugin
   if (ja3_idx >= 0) {
-    TSError(PLUGIN_NAME, "TSRemapInit(): JA3 configured as both global and remap. Check plugin.config.");
+    TSError(PLUGIN_NAME, "JA3 configured as both global and remap. Check plugin.config.");
```

Comment:
Let's add `PLUGIN_NAME` here. I think there's four `TSError` messages in the plugin? Let's add `[ja3_fingerprint]` to each.